### ISSUE TITLE
ServerlessError: You cannot specify a name when defining more than one rate expression

### ISSUE THREAD

**Issue by kitto006** [juanjoDiaz/serverless-plugin-warmup#285]
This issue was originally opened here https://github.com/serverless/serverless/issues/9974
There is an incompatibility issue between core Serverless framework after the Sept 8, 2021 commit (https://github.com/serverless/serverless/commit/9f0bc689cc2ac3e53b4db665b899e1446ac37456) and the serverless-plugin-warmup.

The serverless-plugin-warmup plugin uses the "custom" section.
The old syntax allowed for custom naming of the cloudwatch schedule trigger rule

```yaml
custom:
  warmup:
    default:
      enabled: true # Whether to warm up functions by default or not
      name: warmer-${self:service}-${self:custom.stage}
      events:
        - schedule: 
            name: rule-warmer-${self:service}-${self:custom.stage}
            description: rule-warmer-${self:service}-${self:custom.stage}
            rate: rate(5 minutes)
```

After the Sept 8, 2021 update the below syntax would work, but not the above.
```yaml
custom:
  warmup:
    default:
      enabled: true # Whether to warm up functions by default or not
      name: warmer-${self:service}-${self:custom.stage}
      events:
        - schedule: 'rate(5 minutes)'
```

**Comment by juanjoDiaz** [comment]
Hi @kitto006 ,

Thanks for reporting!

Serverless framework shouldn't introduce breaking changes... specially a week before going to v3 where it could have gone without breaking anything...

I'll try to work with the serverless guys to solve this.
In the meantime you can just use their new syntax:
```yaml
custom:
  warmup:
    default:
      enabled: true # Whether to warm up functions by default or not
      name: warmer-${self:service}-${self:custom.stage}
      events:
        - schedule: 
            name: rule-warmer-${self:service}-${self:custom.stage}
            description: rule-warmer-${self:service}-${self:custom.stage}
            rate:
              - rate(5 minutes)
```

**Comment by kitto006** [comment]
@juanjoDiaz, thanks I will use the new syntax.
This change to array and the new syntax hasn't been mentioned yet in the docs https://www.serverless.com/plugins/serverless-plugin-warmup

**Comment by juanjoDiaz** [comment]
Resolved in v6.1.0
